packages/fxa-auth-db-mysql/lib/db/mysql.js

```diff
@@ -1364,6 +1364,16 @@ module.exports = function (log, error) {
       )
   }
 
+  const READ_SECURITY_EVENTS = 'CALL readSecurityEvents_1(?)'
+  MySql.prototype.getSecurityEvents = function (uid) {
+    return this.read(READ_SECURITY_EVENTS, [uid])
```

Comment:
We'll need a corresponding method [for the in-memory database](https://github.com/mozilla/fxa/blob/9ca5c4057cde5da1e2866cb9515e88bb18e5fb2b/packages/fxa-auth-db-mysql/lib/db/mem.js#L1070) too. The in-memory database is what we normally use while developing and allows us to rapidly test the API w/o requiring MySQL.